README.md

```diff
@@ -73,6 +73,7 @@ Configurable properties :
 | `oidc.audience` | | Optional : audience to validate. Must be the same as the token's `aud` field |
 | `oidc.username-claim` | `preferred_username` | Claim to be used as user id. Must conform to [RFC 1123](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names) |
 | `oidc.groups-claim` | `groups` | Claim to be used as list of user groups. |
+| `oidc.extra-params` | | Optional : map of params to be used by client. e.g : `{ kc_locale: 'fr', kc_idp_hint: 'google'}` |
```

Comment:
```suggestion
| `oidc.extra-params` | | Optional : map of params to be used by client. e.g : `{ kc_idp_hint: 'google'}` |
```

`kc_locale` is not a great example since onyxia ui adds it already

Reply:
A suggestion for example can be `prompt: 'consent'`  (which we will use this feature for)